code/ui/components/src/tabs/tabs.stories.tsx

```diff
@@ -262,18 +255,20 @@ export const StatefulDynamicWithSelectedAddon = {
   play: async (context) => {
     await StatefulDynamicWithOpenTooltip.play(context);
 
-    const popperContainer = screen.getByTestId('tooltip');
-    const tab4 = getByText(popperContainer, 'Tab title #4', {});
-    fireEvent(tab4, new MouseEvent('click', { bubbles: true }));
-    await waitFor(() => screen.getByText('CONTENT 4'));
+    await waitFor(async () => {
+      const popperContainer = await screen.findByTestId('tooltip');
+      const tab4 = await findByText(popperContainer, 'Tab title #4', {});
+      fireEvent(tab4, new MouseEvent('click', { bubbles: true }));
+      await waitFor(() => screen.findByText('CONTENT 4'));
+    });
```

Comment:
I don't hope you need the whole thing wrapped in `waitFor`, since [`findBy` queries already use `waitFor` internally](https://testing-library.com/docs/queries/about/#types-of-queries).


```suggestion
    const popperContainer = await screen.findByTestId('tooltip');
    const tab4 = await findByText(popperContainer, 'Tab title #4', {});
    fireEvent(tab4, new MouseEvent('click', { bubbles: true }));
    await screen.findByText('CONTENT 4');
```

Reply:
I think i did this to get a better view of what is going on in the interactions addon 😅